docs/src/Upsell.doc.js

```diff
@@ -39,6 +39,16 @@ card(
         ],
         href: '',
       },
+      {
+        name: 'image',
```

Comment:
Not convinved this is the best prop name, as it's used for both images and icons. Open to suggestions here!

Reply:
Hey Erica, played around with the code. These are some thoughts and comments. https://www.loom.com/share/ce017b46c28a45c6984a563d38788d9c